tutorial/about-tutorial.md

```diff
@@ -1,101 +1,58 @@
-# Tutorial articles
+# The tutorial template 
 
-Tutorials are learning-oriented lessons that guide you through a series of steps to complete a project.
-They are essential to give new users the feeling that they can quickly achieve something with the product or application.
```

Comment:
In case you are unfamiliar with it, there are two influential articles which have influenced some of the thinking here:
* https://documentation.divio.com/
* https://ffeathers.wordpress.com/2018/10/08/what-is-a-quickstart-to-you/
In a deep dive doc, it would be worth referencing these sources.